src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -76,6 +76,8 @@ class Container implements IntrospectableContainerInterface
     protected $scopeStacks;
     protected $loading = array();
 
+    private static $underscoreMap = array('_' => '', '.' => '_', '\\' => '_');
```

Comment:
It's faster and private. But I just made it non static, that won't make a significant difference.